roles/deploy/tasks/main.yml

```diff
@@ -167,7 +148,7 @@
   command: bash -lc "bundle exec rake db:schema:load RAILS_ENV={{ rails_env }}"
```

Comment:
I'm wondering if we should separate *initial deploy* from following *deploy*s. One mistake in the `when` clause and we wipe out production data. But I guess that's not the Ansible way of doing things. We just have to test things well and you seem to be doing that. :-)

Reply:
Yes, I was planning on doing something like that after the next few bits are merged. The database creation is now outside of the deploy script's responsibilities, and that's a big step forward to being able to untangle things.